client/api/models/index.js

```diff
@@ -12,7 +12,7 @@ const config = require(__dirname + "/../database/database.js")[env];
 const db = {};
 
 let sequelize;
-
+console.log(config);
```

Comment:
small non-blocking nit: i think we should start moving towards cleaning code up so we should only `console.log` if necessary for future production debuggiung